frontend/ui/widget/container/inputcontainer.lua

```diff
@@ -52,7 +52,8 @@ end
 
 function InputContainer:paintTo(bb, x, y)
     if not self.dimen then
-        self.dimen = self[1]:getSize()
+        local content_size = self[1]:getSize()
```

Comment:
If we are sure self.dimen is always a Geom object (which it really should), we can use Geom:copy() method.